### PR TITLE
adds MERN scripts that may need to be modified

### PR DESCRIPTION
**Pull request by sammiefrog**
#6 added mern scripts to server package.json using yarn

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,10 +3,21 @@
...
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "yarn run clean:reports && yarn run test:server && yarn run test:client",
+    "test:server": "cd server && nyc mocha 'tests/**/*.js' --recursive",
+    "test:client": "cd client && yarn test:coverage",
+    "dev:start:server": "nodemon app.js",
+    "dev:start:client": "cd client && yarn start",
+    "dev:start": "concurrently \"yarn run dev:start:server\" \"yarn run dev:start:client\"",
+    "install": "cd client && yarn install",
+    "start:prod": "nodemon app.js",
```
Comment (krab7191) [inline package.json]: We don't want to use nodemon for production, lets just do `node app.js` for now.

**Review by krab7191 — COMMENTED** [review]
Nice! Just a note on deps. You'll need to add nodemon, concurrently, mocha, etc to the devDependencies list. Also lets delete the package-lock.json and just keep using yarn.

File: package.json (hunk 1 above)
Comment (sammiefrog) [reply package.json]: got it! changing now.

**Review by sammiefrog — COMMENTED** [review]
Added the devDependencies and switched production script to node instead of nodemon.

File: package.json [hunk 2]
```diff
@@ -3,10 +3,26 @@
...
+    "concurrently": "^5.3.0",
+    "nodemon": "^2.0.7",
+    "nyc": "^15.1.0",
+    "mocha": "^8.2.1"
+  },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "yarn run clean:reports && yarn run test:server && yarn run test:client",
+    "test:server": "cd server && nyc mocha 'tests/**/*.js' --recursive",
+    "test:client": "cd client && yarn test:coverage",
```
Comment (krab7191) [inline package.json:18]: We'll need a 'test:coverage' script in the client, but this is fine for now, we don't have any tests set up anyway.

**Review by krab7191 — APPROVED** [review]

**Pull request merged**